.github/workflows/tests.yml

```diff
@@ -24,6 +24,9 @@ defaults:
 permissions:
   contents: write
 
+env:
+  GITHUB_REPOSITORY: janopl/ddev-kibana
```

Comment:
I'm pretty sure this isn't required.

Reply:
Unfortunately this parameter is required and also the path to the folder with tests, action has checkout inside but it allows me to split into several jobs

Reply:
Hmm, I didn't use anything like that in https://github.com/ddev/ddev-beanstalkd - did I forget something or do something wrong? I was assuming that was provided by GH. He didn't suggest that in https://github.com/julienloizelet/ddev-add-on-test

I guess maybe you're doing something a bit more exotic.